cypress/integration/brewery/TableParameters-files_part4.spec.js

```diff
@@ -57,13 +57,35 @@ describe('Table parameters files standard operations part 4', () => {
     BreweryParameters.getEventsTableCell('timeOfDay', 2).should('have.text', 'evening');
     BreweryParameters.getEventsTableCell('reservationsNumber', 3).should('have.text', '220');
     BreweryParameters.getEventsTableCell('online', 4).should('have.text', 'false');
+
+    BreweryParameters.editEventsTableStringCell('reservationsNumber', 3, '50').should('have.text', '50');
+    ScenarioParameters.getSaveButton().should('be.visible');
+    ScenarioParameters.getParametersDiscardButton().should('be.visible');
+    ScenarioParameters.discard();
+    ScenarioParameters.getSaveButton().should('not.exist');
+    ScenarioParameters.getParametersDiscardButton().should('not.exist');
+    BreweryParameters.getEventsTableCell('reservationsNumber', 3).should('have.text', '220');
+
+    BreweryParameters.editEventsTableStringCell('reservationsNumber', 3, '77').should('have.text', '77');
+    ScenarioParameters.getParametersDiscardButton().should('be.visible');
+    ScenarioParameters.getSaveButton().should('be.visible');
```

Comment:
There is no need to check both buttons every time, the behavior of these buttons is probably already checked in another test. For the bug being fixed in this branch, we simply need to check the state of one button (e.g. the "save" button) :slightly_smiling_face:

Reply:
It's indeed a bit overzealous.